### PR TITLE
server: print welcome msg first when starting

### LINKED ISSUE #6702: Welcome message are not the first message when starting anymore
## Bug Report

I expect seeing "Welcome to TiKV" at the startup messages, but seeing configuration check and connecting to PD instead. This may not be convenient for diagnosticsing problem. If it keeps panicing during connecting to PD, it may not be easy to find it out as we used to check "Welcome" message first.

### What version of TiKV are you using?
Latest master

### What operating system and CPU are you using?
CentOS

### Steps to reproduce
systemctl start tikv-port

### What did you expect?

```
[2020/02/26 10:55:10.727 +00:00] [INFO] [lib.rs:88] ["Welcome to TiKV"]
[2020/02/26 10:55:10.727 +00:00] [INFO] [lib.rs:90] []
[2020/02/26 10:55:10.727 +00:00] [INFO] [lib.rs:90] ["Release Version:   4.1.0-alpha"]
```

### What did happened?
```
[2020/02/26 09:10:07.141 +00:00] [INFO] [config.rs:514] ["kernel parameters"] [value=32768] [param=net.core.somaxconn]
[2020/02/26 09:10:07.141 +00:00] [INFO] [config.rs:514] ["kernel parameters"] [value=0] [param=net.ipv4.tcp_syncookies]
[2020/02/26 09:10:07.141 +00:00] [INFO] [config.rs:514] ["kernel parameters"] [value=0] [param=vm.swappiness]
[2020/02/26 09:10:07.141 +00:00] [INFO] [util.rs:425] ["connecting to PD endpoint"] [endpoints=127.0.0.1:2379]
[2020/02/26 09:10:07.143 +00:00] [INFO] [<unknown>] ["New connected subchannel at 0x7f98ac04b150 for subchannel 0x7f98abc21000"]
[2020/02/26 09:10:07.156 +00:00] [INFO] [server.rs:244] ["connect to PD cluster"] [cluster_id=6797395620193450052]
[2020/02/26 09:10:07.224 +00:00] [INFO] [server.rs:759] ["beginning system configuration check"]
[2020/02/26 09:10:07.224 +00:00] [INFO] [lib.rs:88] ["Welcome to TiKV"]
[2020/02/26 09:10:07.224 +00:00] [INFO] [lib.rs:90] []
[2020/02/26 10:55:10.727 +00:00] [INFO] [lib.rs:90] ["Release Version:   4.1.0-alpha"]
```

### PR DESCRIPTION
**Pull request by NingLin-P**
Signed-off-by: linning <linningde25@gmail.com>

###  What have you changed?

Make welcome message be the first message when starting. fix #6702 

###  What is the type of the changes?

- Bugfix (a change which fixes an issue)

### CODE REVIEW

File: cmd/src/server.rs [hunk 1]
```diff
@@ -66,6 +66,9 @@ use tikv_util::{
 /// Run a TiKV server. Returns when the server is shutdown by the user, in which
 /// case the server will be properly stopped.
 pub fn run_tikv(config: TiKvConfig) {
+    // Print version information.
+    tikv::log_tikv_info();
```
Comment (overvenus) [inline cmd/src/server.rs]: Please make sure logger is initialized.

File: cmd/src/server.rs [hunk 2]
```diff
@@ -66,8 +66,18 @@ use tikv_util::{
 /// Run a TiKV server. Returns when the server is shutdown by the user, in which
 /// case the server will be properly stopped.
 pub fn run_tikv(config: TiKvConfig) {
+    // Sets the global logger ASAP.
+    // It is okay to use the config w/o `validate()`,
+    // because `initial_logger()` handles various conditions.
+    // TODO: currently the logger config has to be provided
```
Comment (BusyJay) [inline cmd/src/server.rs]: It needs to be supported in this PR. It's not backward compatible.

Comment (NingLin-P) [reply cmd/src/server.rs]: It is OK, the order of code executing are just like before except `tikv::log_tikv_info()` moving forward.

Comment (BusyJay) [reply cmd/src/server.rs]: In the past, logger can be configured via command line or configuration file. I think the "before" should be version 3.0 and older.

Comment (NingLin-P) [reply cmd/src/server.rs]: Sorry, the comment was misleading, what that want to said was the logger's config can not get from PD like other config, but it still can be configured via command line or configuration file, I will fix the comment.

**Pull request merged**